### DISCUSSION TITLE
How to add the GUID to Gunicorn access logs?

### DISCUSSION THREAD

**Discussion by axieum** [snok/django-guid#83] [Q&A] [answered]
For anyone running their Django app behind Gunicorn, you'll notice it handles access logging of requests. How should we go about adding the GUID into said access logs?

**Comment by axieum** [comment, marked as answer]
By looking at Gunicorn's configurable [`access_log_format`](https://docs.gunicorn.org/en/latest/settings.html#access-log-format), you can access request/response headers: `%({correlation-id}o)s`

**Reply by JonasKs** [reply]
Yup!😊
